staging/src/k8s.io/apiextensions-apiserver/artifacts/customresource-01/noxu-resource-definition.yaml

```diff
@@ -1,13 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: noxus.mygroup.example.com
-spec:
-  group: mygroup.example.com
-  version: v1alpha1
-  scope: Namespaced
-  names:
-    plural: noxus
-    singular: noxu
-    kind: Noxu
```

Comment:
these noxus finished their k/k journey 🙂

Reply:
Haha, no they are still being used in integration tests :smile: